var/spack/repos/builtin/packages/pocl/package.py

```diff
@@ -55,15 +57,18 @@ class Pocl(CMakePackage):
     provides('opencl@:2.0')
 
     depends_on("cmake @2.8.12:", type="build")
-    depends_on("hwloc")
+    depends_on("hwloc@:1.99.99", when="@:1.1.99")
+    depends_on("hwloc", when="@1.2:")
```

Comment:
I would just say:
```python
depends_on("hwloc")
depends_on("hwloc@:1.99.99", when="@:1.1.99")
```
This is equivalent, but works around a bug where `spack spec pocl ^hwloc` will tell you that `pocl` doesn't depend on `hwloc`.

Reply:
Done!